ministryofjustice/correspondence_tool_staff
# CT-2673, CT-2674, CT-2676 cabinet office report

## Description

This PR is to fix the figure issues around cabinet office report

## Self-review checklist


### Screenshots


### Related JIRA tickets

https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=25&modal=detail&selectedIssue=CT-2673
https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=25&modal=detail&selectedIssue=CT-2674
https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=25&modal=detail&selectedIssue=CT-2676
### Deployment


### Manual testing instructions

1. Login in as London User
2. Click reports tab from top menu
3. Click "Download report" from Cabinet Office report part
4. Check the figure
